src/modules/static-pages/about/components/section/style.module.css

```diff
@@ -0,0 +1,38 @@
+.about-home_section {
+    margin-top: var( --section-top-spacing);
+    margin-bottom: var(--section-bottom-spacing);
+    background-color: var(--layout-primary);
+ }
+.about-home_section .about-home_Heading{
+    font-size:var(--heading-2-font-size);
+    color: var( --layout-secondary);
+    padding: var( --section-heading-spacing);
+    letter-spacing: 0.12rem;
+    font-family: var( --heading-font-family);
+    text-align: center;
+}
+
+@media (max-width: 991px) {
+    .about-home_section  .about-home_Info  {
+        padding-top: 3rem;
+    } 
+}
+.about-home_section  .about-home_Info .about-home_lead {
+    color: var( --text-secondary);
```

Comment:
No need to set the color of the paragraph as it has already been set, and it should be white not blue.